config/scheduler-chromeos.yaml

```diff
@@ -1039,3 +1039,15 @@ scheduler:
 
   - job: wifi-basic-arm64-qualcomm
     <<: *test-job-arm64-qualcomm
+
+  - job: wifi-basic-cros-kernel-arm64-mediatek
+    <<: *test-job-arm64-mediatek
```

Comment:
I think this should be `test-job-arm64-mediatek-cros-kernel` - same for the others below:
- test-job-x86-amd -> test-job-x86-amd-cros-kernel
- test-job-x86-intel -> test-job-x86-intel-cros-kernel
- test-job-arm64-qualcomm -> test-job-arm64-qualcomm-cros-kernel

Reply:
Yes, thanks. Will update and test it.